### DISCUSSION TITLE
Make UR matter in scores.

### DISCUSSION THREAD

**Discussion by LeviathanGR** [ppy/osu#21866] [Ideas] [answered]
As the title says, I think it would be much better for the leaderboards especially like if someone has the same score with another player the one with the better UR gets a better position in the leaderboard.
This could be a good addition to the PP algorithm.

**Comment by Rekunan** [comment]
That would mean 2 SS scores have the possibility of awarding different acc pp, which I'm not too sure is logical.

**Reply by LeviathanGR** [reply]
Maybe not affecting PP, just the score so the real better score (better UR) stays on the top.

**Reply by Rekunan** [reply]
That just shifts from acc pp to acc score, which again, I'm not too sure is logical.

**Comment by peppy** [comment, marked as answer]
We've discussed adding more judgements to osu! (similar to osu!mania), but considering a full floating point range of UR is not going to happen. It's too much.
